c/verifythis/Makefile

```diff
@@ -0,0 +1,12 @@
+LEVEL := ../
+
+GCC_WARNINGS := \
+	-Wno-bool-compare \
```

Comment:
Is it possible to avoid these and the clang warnings instead of suppressing them?